guides/source/caching_with_rails.md

```diff
@@ -30,117 +33,115 @@ config.action_controller.perform_caching = true
 
 ### Page Caching
 
-Page caching is a Rails mechanism which allows the request for a generated page to be fulfilled by the webserver (i.e. Apache or NGINX), without ever having to go through the Rails stack at all. Obviously, this is super-fast. Unfortunately, it can't be applied to every situation (such as pages that need authentication) and since the webserver is literally just serving a file from the filesystem, cache expiration is an issue that needs to be dealt with.
+Page caching is a Rails mechanism which allows the request for a generated page
+to be fulfilled by the webserver (i.e. Apache or NGINX) without ever hitting
+your Rails app. While this is super fast it can't be applied to every situation
+(such as pages that need authentication). Also, because the webserver is
+serving a file directly from the filesystem you will need to implement cache
+expiration.
 
 INFO: Page Caching has been removed from Rails 4. See the [actionpack-page_caching gem](https://github.com/rails/actionpack-page_caching).
 
 ### Action Caching
 
-Page Caching cannot be used for actions that have before filters - for example, pages that require authentication. This is where Action Caching comes in. Action Caching works like Page Caching except the incoming web request hits the Rails stack so that before filters can be run on it before the cache is served. This allows authentication and other restrictions to be run while still serving the result of the output from a cached copy.
+Page Caching cannot be used for actions that have before filters – for example,pages that require authentication. This is where Action Caching comes in. Action Caching works like Page Caching except the incoming web request hits the Rails stack so that before filters can be run prior to the cache being served. This allows authentication and other restrictions to be run while still serving the result of the output from a cached copy.
```

Comment:
can you clarify the reason for changing 'before the cache is served'? The new version should read 'prior to the cache's being served'